### PR TITLE
fix: impact rendering

### PR DESCRIPTION
**Pull request by zzmp**
- Fixes price impact rendering, which was broken when attempting to render the PriceImpact object 😱 
- Fixes the fade-out gradient in WebKit by using `transparent` explicitly (instead of a hex version)

### CODE REVIEW

**Comment by JFrankfurt** [comment]
not super happy about the switch from `priceImpact` to just `impact` (though I see it wasn't all `priceImpact` initially)

**Review by JFrankfurt — APPROVED** [review]

**Comment by zzmp** [comment]
> not super happy about the switch from `priceImpact` to just `impact` (though I see it wasn't all `priceImpact` initially)

I just wanted it to be consistent, and generally go for more concise wording. Happy to do either way. Also wouldn't be offended if you put in a PR to update all the names to `priceImpact`, as long as it stays consistent.
